app/src/docs/v1.api-spec.yaml

```diff
@@ -38,6 +38,11 @@ tags:
     externalDocs:
       url: >-
         https://github.com/bcgov/common-object-management-service/wiki/Endpoint-Notes#permission
+  - name: Sync
+    description: Operations for syncing existing buckets and objects.
```

Comment:
Operations for syncing existing buckets and objects. Ensures that external S3 modifications are eventually correctly tracked by COMS.